crypto/chacha/asm/chacha-x86_64.pl

```diff
@@ -761,12 +755,12 @@ sub SSSE3_lane_ROUND {
 	movdqa		$xb2,0xa0-0x100(%rcx)
 	movdqa		$xb3,0xb0-0x100(%rcx)
 
-	pshufd		\$0x00,$xt3,$xt0	# "$xc0"
-	pshufd		\$0x55,$xt3,$xt1	# "$xc1"
+	pshufd		\$0x00,$xt3,$xt0	# "\$xc0"
```

Comment:
This is kind of silly. I'm not sure if you'd prefer the sigil removed altogether or the variables declared earlier or whatever. Note that $xaN (below) isn't actually a variable.